src/tests/simulation/src/main.rs

```diff
@@ -169,10 +170,15 @@ async fn main() {
     if let Some(count) = args.sqlsmith {
         cluster
             .run_on_client(async move {
+                let seed = env::var("MADSIM_TEST_SEED")
+                    .unwrap()
+                    .parse::<u64>()
+                    .unwrap();
```

Comment:
The seed can be obtained by:
```suggestion
                let seed = madsim::runtime::Handle::current().seed();
```

Reply:
Thanks!